### DISCUSSION TITLE
SEAWAT code request

### DISCUSSION THREAD

**Discussion by Sakura223Leslie** [modflowpy/flopy#2252] [Q&A] [answered]
I can not found the code to use SEAWAT with flopy. Could you give me some code example, thanks.

**Comment by mbakker7** [comment, marked as answer]
Have you checked out these examples: https://flopy.readthedocs.io/en/latest/examples.html#mt3d-and-seawat-examples
Especially this one: https://flopy.readthedocs.io/en/latest/Notebooks/seawat_henry_example.html

**Reply by wpbonelli** [reply]
Should the SEAWAT Henry example be moved into the MT3D and SEAWAT section? Looks like neither example in that section uses SEAWAT at the moment. Might have been my mistake when setting up the notebook gallery a while ago.

**Reply by mbakker7** [reply]
Yes that should be moved. I thought we had more seawat examples in the past but I may be mistaken. 
Note that there are quite a few MODFLOW 6 examples with density flow!

**Reply by Sakura223Leslie** [reply]
So I guess can not found the code using the SEAWAT at this moment? Could you recommend use MODFLOW6 or SEAWAT for seawater intrusion? Thanks for your help.
